twilio-authenticator-sample/src/main/java/com/twilio/authenticatorsample/appdetail/AppDetailFragment.java

```diff
@@ -1,40 +1,38 @@
-package com.twilio.authenticatorsample.main;
+package com.twilio.authenticatorsample.appdetail;
 
-import android.content.Intent;
+import android.app.ActionBar;
```

Comment:
I think some of the imports are not needed anymore. Could you clean them? in Android Studio go to Code->Optimize Imports